### ISSUE TITLE
Messages that contain media (Pics, vids, stickers, voice messages, etc.) are blank, even when there is text attached to the message.

### ISSUE THREAD

**Issue by CheeseburgerWithFries** [pigpagnet/save-your-telegram-history#10]
When I highlight and copy-paste a message directly from the web.telegram interface, I get this output:

>[10:54:27 AM] User:
>[Photo]
>This is a kitty.

Where "This is a kitty." is the caption for the photo sent in the same message.

This is the same message, as output by the Save Telegram Chat History extension:

> 08.12.2016 10:54:27, User(you): 

This problem exists with all media. Telegram uses a consistent format for all media files with [Video file], [Voice message], [GIF] and [🍗 Sticker] among them. The sticker output uses whatever emoji/emoticon that sticker has been paired to in Telegram's system. This format is used on all platforms, with the downloaded Windows app having a similar output. None of these output any message text in the extension.

If this could be fixed, I'd really appreciate it. Not only are you losing part of the conversation when the attached text disappears, but it's confusing too when you run across a string of 10 blank messages in a row. If those messages all read [Voice message], then at least you'd know what was happening in the conversation.

**Comment by vanchope** [comment]
Fixed. Will be released as part of milestone 2.1. 

Below are some examples:

```
User(you) [11.02.2017 23:23:56]
[[Photo]]Some text along with the photo

User(you) [11.02.2017 23:24:44]
[[Voice Message, 4sec, size 9'295 bytes]]

User(you) [11.02.2017 23:26:14]
[[Contact: A Check  *101#]]

User(you) [11.02.2017 23:26:48]
[[Geo Lat/Long = 20.659324333555123, -11.406255181160704]]

User(you) [11.02.2017 23:27:23]
[[File "collection.log", size 7'191 bytes]]

User(you) [11.02.2017 23:28:12]
[[Video, size 107'968 bytes]]

User(you) [12.02.2017 00:44:54]
[[Webpage]]http://google.com

User(you) [12.02.2017 00:45:23]
[[Webpage]]webpage http://google.com

User(you) [12.02.2017 00:55:59]
⛷

User(you) [12.02.2017 00:56:09]
[[✊ Sticker, size 36'326 bytes]]

User(you) [12.02.2017 01:13:55]
[[GIF, size 18'369 bytes]]
```

**Comment by CheeseburgerWithFries** [comment]
@vanchope Thanks!
